cooked-validators/.gitignore

```diff
@@ -1 +0,0 @@
-cooked-validators.cabal
```

Comment:
Very interesting that this file was committed but ignored!

Reply:
It caused me some painful hours of debugging... because the pre-commit hooks filter out everything that is ignored and I didn't understand why some error were not caught.